.env

```diff
@@ -1,4 +1,4 @@
 REACT_APP_CHAIN_ID="1"
 REACT_APP_GOOGLE_ANALYTICS_MEASUREMENT_ID=""
 REACT_APP_NETWORK_URL="http://127.0.0.1:8545/"
-REACT_APP_NOTIFY_DAPP_ID="4ea9b565-dff8-4415-a65d-21c085a6110c"
\ No newline at end of file
+REACT_APP_NOTIFY_DAPP_ID="abc43422-2242-4292-8223-0a02687d627f"
```

Comment:
@hammeiam what are your thoughts on this change?

Reply:
I don't think we should need to change this - were you runninng into errors with the old one?

Reply:
no, I just made the change right off the bat.

Reply:
I just confirmed that this older key `abc...` this error comes up: 
![image](https://user-images.githubusercontent.com/5507707/134420816-44074705-fd2f-48e7-8d2e-3703594ec41e.png)

it doesn't happen when the key is updated.